core/templates/pages/admin-page/activities-tab/admin-prod-mode-activities-tab.component.ts

```diff
@@ -13,23 +13,19 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive for the activities tab in the admin panel when Oppia
+ * @fileoverview Component for the activities tab in the admin panel when Oppia
```

Comment:
Will it be possible to add a simple test file that ensures this component exists and gets initialized correctly?

@srijanreddy98, Are we allowing simple/empty components without tests?

Reply:
Done.